content/index.md

```diff
@@ -10,30 +10,34 @@ Und? Was wirst du erfinden?
 :::
 
 
-# Willkommen bei den dritten 23. Rüspeler Tüfteltagen!
+# Willkommen bei den vierten 23. Rüspeler Tüfteltagen!
+
+Immer noch Pandemie - trotzdem Bock auf coole Leute. Deswegen wird es die vierten 23. Rüspeler Tüfteltage wieder als Online Livestream geben. Live und in Farbe aus Rüspel! Wir haben Programm und zusammen mit euch wollen wir einen wunderbaren Samstag Abend verbringen:
+
+**Samstag (09.04.2020) ab 17:00 Uhr**
 
-Da nun feststeht, dass es keine 3. 23.Rütüta geben wird, haben wir ein Alternativprogramm auf die Beine gestellt – natürlich online. Dazu streamen wir kleine Vorträge und Workshops am
-**Samstag (09.05.2020) ab 17:00 Uhr**
 **auf**
-**[https://www.twitch.tv/ruetueta/](https://www.twitch.tv/ruetueta/)**
-Wir freuen uns über rege Beteiligung. Du kannst einfach nur zuschauen, oder zusammen mit uns kleine Projekte hacken, zocken oder auch nur mit anderen Leuten ein bisschen schnacken. Die nötige Infrastruktur stellen wir natürlich bereit.
+**[https://www.twitch.tv/chrisfigge](https://www.twitch.tv/chrisfigge)**
+
+Wir freuen uns über rege Beteiligung. Du kannst einfach nur zuschauen oder zusammen mit uns kleine Projekte hacken, zocken oder auch nur mit anderen Leuten ein bisschen schnacken.
+
+## Badge
+
+Es wird diesmal zum aller ersten mal in der Geschichte der 23. Rüspeler Tüfteltagen ein Löt Badge geben. Mit unserem kleinen Wechselblinker Bausatz kannst du dir die rütüta nach Hause hohlen.
```

Comment:
Es wird diesmal zum allerersten Mal in der Geschichte der 23. Rüspeler Tüfteltage ein Löt-Badge geben. Mit unserem kleinen Wechselblinker-Bausatz kannst du dir die RüTüTa nach Hause holen.